src/repositoryConnection/RepositoryConnection.ts

```diff
@@ -84,7 +84,7 @@ export class RepositoryConnection {
 
 		try {
 			const response = await this.octokit.request(
-				"GET /repos/{owner}/{repo}/contents/{path}",
```

Comment:
This is not a redundant path. `contents` is part of the GitHub REST api path to the root of a git project. Removing `contents` from the path breaks the ability to publish multiple files at once.